operators/src/engine/query_processor.rs

```diff
@@ -9,25 +11,14 @@ use geoengine_datatypes::plots::{PlotData, PlotOutputFormat};
 use geoengine_datatypes::raster::Pixel;
 use geoengine_datatypes::{collections::MultiPointCollection, raster::RasterTile2D};
 
-/// An instantiation of an operator that produces a stream of results for a query
-#[async_trait]
-pub trait QueryProcessor: Send + Sync {
-    type Output;
-    async fn query<'a>(
-        &'a self,
-        query: QueryRectangle,
-        ctx: &'a dyn QueryContext,
-    ) -> Result<BoxStream<'a, Result<Self::Output>>>;
-}
-
```

Comment:
I guess there is no need to remove it, we can add another type, cf. https://github.com/geo-engine/geoengine/pull/269

Reply:
We don't use `query` anymore so I guess we don't need the common trait anymore

Reply:
We can use query, you just change it in this PR.

Reply:
but what is the benefit of using a common query method if both the input and the output types are different?